app/models/flow_download.rb

```diff
@@ -13,7 +13,7 @@ def initialize(context, params)
     else
       ','
     end
-    @download_name = "#{@context.country.name}_#{@context.commodity.name}_#{DownloadVersion.current_version_symbol}"
+    @download_name = [@context.country.name, @context.commodity.name, DownloadVersion.current_version_symbol(@context)].reject(&:nil?).join('_')
```

Comment:
`reject(&:nil?)` can be expressed more shortly as `compact`

Reply:
fixed